core/src/test/java/org/elasticsearch/index/engine/InternalEngineTests.java

```diff
@@ -973,13 +973,12 @@ public void testSimpleOperations() throws Exception {
         searchResult.close();
 
         // but, not there non realtime
-        Engine.GetResult getResult = engine.get(newGet(false, doc), searcherFactory, (onRefresh) -> fail("shouldn't have a refresh"));
+        Engine.GetResult getResult = engine.get(newGet(false, doc), searcherFactory);
```

Comment:
I wonder if it is worth keeping the assertions about whether or not we refreshed. Certainly up to you.